config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```
@@ -172,6 +172,50 @@ periodics:
           cpu: 7300m
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api
-    testgrid-tab-name: capi-e2e-main-1-21-latest
+    testgrid-tab-name: capi-e2e-main-1-21-1-22
+    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: "2"
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-22-latest-main
+  interval: 24h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-1.21
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
+          value: "stable-1.22"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "ci/latest-1.23"
+        - name: ETCD_VERSION_UPGRADE_TO
+          value: "3.4.13-0"
```

According to https://github.com/EricWvi/kubernetes/blob/332488aae4399dde6f983a429d564fa8aa42d77e/cmd/kubeadm/app/constants/constants.go#L460-L472 this should be
```suggestion
          value: "3.5.0-0"
```
Could you kindly change this at L162 and in pull-cluster-api-e2e-workload-upgrade-1-22-latest-main as well?

---

Thank you @fabriziopandini, done.